testapi.pm

```diff
@@ -859,7 +859,10 @@ sub assert_shutdown {
             sleep($timeout);
         }
         if ($is_shutdown) {    # -1 counts too
-            $autotest::current_test->take_screenshot('ok');
+                               # With svirt backend the VNC connection is dead, can't take a screenshot
```

Comment:
please move the comments a bit so that perltidy does not align them. this looks silly :)